proxy/tests/graphql_mutation.rs

```diff
@@ -186,7 +193,7 @@ fn register_project() {
         let query = "mutation($projectName: String!, $orgId: String!) {
                         registerProject(projectName: $projectName, orgId: $orgId) {
                             messages {
-                                ... on ProjectRegistration {
+                                ... on ProjectRegistrationMessage {
                                     projectName,
                                     orgId
                                 }
```

Comment:
Can you add the `registered` info to the response here?

Reply:
Here we get back a transaction, which is not the actual project. But it's a good point we should query for it when we test for retrieving a project.